### ISSUE TITLE
Make it possible to filter on GSSP tokens in RA token overview & export

### ISSUE THREAD

**Issue by phavekes** [OpenConext/Stepup-RA#373]
[This issue is imported from pivotal - Originaly created at Dec 4, 2017 by Michiel Kodde](https://www.pivotaltracker.com/story/show/153357335)

Previous gssp tokens are in the list (biometric & tiqr) configuration driven GSSP\'s are not yet listed in this filtering listing. This might be a hardcoded list.

~~In addition, hardcoded references to tiqr or other GSSP devices should be checked for inconsistencies/bugs. As some code still relies on hardcoded gssp configuration.~~

Moved to chore #155975614

**Comment by phavekes** [comment]
A solution was chosen that forks on the previous GSSP configuration solution. 

In short, the gssp label that is shown on the form select list can be configured in the view_config section of the `samlstepupproviders_parameters.yml`. The changes can be found in [PR#149](https://github.com/OpenConext/Stepup-RA/pull/149)

The readme containing the gssp installation instructions have been updated in this [OpenConext deploy pr](https://github.com/OpenConext/Stepup-Deploy/pull/50) (Michiel Kodde - Mar 9, 2018)

**Comment by phavekes** [comment]
To address the two extra tasks added to this story a couple of new PR\'s have been opened.
- [Middleware](https://github.com/OpenConext/Stepup-Middleware/pull/214)
- [Stepup-bundle](https://github.com/OpenConext/Stepup-bundle/pull/46) (will result in release of a new release 3.1.1.

The new release of the Stepup bundle will be installed into the Stepup applcations (RA, SS, GW, MW). PR\'s achieving that goal will be opened once the Stepup-bundle PR#46 is merged. (Michiel Kodde - Mar 13, 2018)
